app/javascript/angular/dashboard/widget-wrapper.js

```diff
@@ -26,23 +25,49 @@ ManageIQ.angular.app.component('widgetWrapper', {
     vm.$onInit = function() {
       vm.divId = `w_${vm.widgetId}`;
       vm.innerDivId = `dd_w${vm.widgetId}_box`;
+      vm.refreshWidgetHTML(false);
+      vm.parsedButtons = JSON.parse(vm.widgetButtons);
+      const refreshButton = vm.parsedButtons.find(ob => ob.id.includes("refresh"));
```

Comment:
`ob => ob.refresh` ?

(Why match against the id when you already have that `refresh: true` (https://github.com/ManageIQ/manageiq-ui-classic/pull/6334/files#diff-8ff9fd36a766950b59f7184e53f875e1R92).)

Reply:
🤦‍♀ Fixed.